idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -1071,7 +1052,9 @@ def run_additional_steps():
         execute_command("echo This file is deprecated since pipeline version 1.5.  Please use {new} instead. > {deprecated}".format(
             deprecated=result_dir(DEPRECATED_BOOBYTRAPPED_COMBINED_JSON_OUT),
             new=MULTIHIT_COMBINED_JSON_OUT))
-        execute_command("aws s3 cp --quiet %s/%s %s/" % (RESULT_DIR, DEPRECATED_BOOBYTRAPPED_COMBINED_JSON_OUT, SAMPLE_S3_OUTPUT_PATH))
+        src = RESULT_DIR + "/" + DEPRECATED_BOOBYTRAPPED_COMBINED_JSON_OUT
```

Comment:
We can actually delete this soon, as the comment suggests.

Reply:
wait should I delete this whole block?